spec/models/email_processor_spec.rb

```diff
@@ -94,17 +94,37 @@
 
     context "when the entry is a response to yesterday's email" do
       it "sets the entry date to yesterday's date" do
```

Comment:
@chrishunt what do you think of making this a more general description? This is really testing any email from earlier in the same year – there's no special case for yesterday.

Might also want to change the `Timecop.freeze` date to be more than one day off...